package.json

```diff
@@ -32,13 +32,13 @@
   "author": "Apache Software Foundation",
   "license": "Apache-2.0",
   "devDependencies": {
-    "eslint": "^4.0.0",
-    "eslint-config-semistandard": "^11.0.0",
-    "eslint-config-standard": "^10.2.1",
-    "eslint-plugin-import": "^2.7.0",
-    "eslint-plugin-node": "^5.1.0",
-    "eslint-plugin-promise": "^3.5.0",
-    "eslint-plugin-standard": "^3.0.1",
+    "eslint": "^5.2.0",
+    "eslint-config-semistandard": "^12.0.1",
+    "eslint-config-standard": "^12.0.0-alpha.0",
```

Comment:
Is there a reason that this one has to be on `@next`? 
Could it be `eslint-config-standard@^11.0.0` as seen in the other evaluation PR #389 while the rest were updated?

I don't see any problems with updating all of them.

Reply:
eslint-config-standard@next was needed to resolve an ugly warning message from eslint@5, details are now in the description

Reply:
FYI `^12.0.0-alpha.0` means that we would use the -alpha version until the publish a newer version, such as -beta, or finally publish 12.0.0 version